src/SayMore/UI/ComponentEditors/SessionBasicEditor.cs

```diff
@@ -619,5 +626,123 @@ private void HandlePeopleEditClick(object sender, EventArgs e)
 				break;
 			}
 		}
+
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		/// Method to save the new AccessProtocol node and update the existing AccessProtocol node when option get changed
+		/// </summary>
+		/// <param name="accessCode">Previous AccessCode</param>
+		/// <returns>New AccessCode</returns>
+		/// ------------------------------------------------------------------------------------
+		private string HandleAccessProtocols(string accessCode)
```

Comment:
The word "Handle" as the first word in a method name normally signals that the method is some kind of event handler. I think it's best to reserve it for that purpose. Neither this method name nor the description help me understand quickly what it's purpose is.